src/stock-adjustment-mobile/edit-product-page/edit-product-page.jsx

```diff
@@ -0,0 +1,315 @@
+            .value();
+
+        return _.map(lots, lot => ({ name: formatLot(lot), value: lot }));
+    };
+
+    const renderLotSelect = (fieldName, product, v) => {
+        const options = getLotsOptions(product);
+        const noOptions = !options?.length;
+        return (
+            <SelectField
+                name={`${fieldName}.lot`}
+                label="Lot Code"
+                options={options}
+                objectKey="id"
+                defaultOption={noOptions ? 'Product has no lots' : 'No lot defined'}
+                disabled={true}
+                containerClass='field-full-width'
+            />
+        );
+    };
+
+    return (
+        <div style={{marginBottom: "40px"}}>
+            <Form
+                initialValues={{ items: productToEdit?.items ?? [{}] }}
+                onSubmit={onSubmit}
+                validate={validate}
+                mutators={{ ...arrayMutators }}
+                decorators={[decorator]}
+                render={({ handleSubmit, values, invalid }) => (
+                    <form className="form-container" onSubmit={handleSubmit}>
+                        <FieldArray name="items">
+                            {({ fields }) => (                       
+                                <div className="form-container">
+                                    <div className="page-header-responsive">
+                                        <div id="header-wrap" style={{marginBottom: "24px"}}>
+                                            <h2 id="product-add-header">{userHomeFacility.code} - {userHomeFacility.name} - {program.programName}</h2>
+                                            <div className="button-inline-container">
+                                                <i 
+                                                    className="fa fa-times fa-3x" 
+                                                    aria-hidden="true"
+                                                    onClick={() => cancel()}
+                                                />
+                                            </div>
+                                        </div>
+                                    </div>
+                                    <div className="form-body">
+                                        {fields.map((name, index) => (
+                                            <div key={name}>
+                                                <SelectField
+                                                    required
+                                                    name={`${name}.product`}
+                                                    label="Product"
+                                                    options={productOptions}
+                                                    objectKey={[0, 'orderable', 'id']}
+                                                    containerClass='field-full-width'
+                                                    disabled={true}
```

Comment:
Same as above - you cas use `disabled` instead of `disabled={true}`. There is a perfect example here - look at `required`.